src/backends/torch/torchlib.cc

```diff
@@ -536,17 +593,25 @@ namespace dd
     try
       {
         inputc.transform(ad);
+
+        _module._native = std::shared_ptr<NativeModule>(
+            NativeFactory::from_template(_template, _template_params, inputc));
+
         if (_module._native)
+          if (!this->_mlmodel._native.empty())
+            torch::load(_module._native, this->_mlmodel._native, _device);
+
+        if (_module._graph)
```

Comment:
I would recommend that anything that depends on values from within `_module` do actually happen in the `TorchModule` class. This is to avoid spreading the code and complexifying torchlib itself.

Reply:
ok, done